<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgReasonToVisitEdit</class>
 <widget class="QDialog" name="dlgReasonToVisitEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ledName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkIndependent">
     <property name="text">
      <string>Independent from the local studio.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btbButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>accepted()</signal>
   <receiver>dlgReasonToVisitEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>dlgReasonToVisitEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
